### DISCUSSION TITLE
double Y axis

### DISCUSSION THREAD

**Discussion by al3xanderSL** [PHPOffice/PhpSpreadsheet#4005] [Q&A]
Hii,  I'm using your library to make a chart in excel. The point is, I want to create a chart with 2 Y axis one on each side and a X axis, I have already done the whole process to achieve it, except the second Y axis to show another dataseries, which is quite different to the first dataseries.Then, is there any method to do it in the library that maybe I haven't seen it? or this feature is not enable for this library.

The target I want to reach, is something like this: 

![goal](https://github.com/PHPOffice/PhpSpreadsheet/assets/150929431/ef608023-93c2-4b05-af14-080babfbf2f3)

**Comment by oleibman** [comment]
We do not yet support a secondary axis. It has been requested before. I haven't yet made any traction on how we might go about supporting it.
